auth_otp/models.py

```
@@ -0,0 +1,438 @@
+    
+    def save(self, *args, **kwargs):
+        if not self.otp_code:
+            self.otp_code = self.generate_otp_code()
+        
+        if not self.expires_at:
+            self.expires_at = timezone.now() + timedelta(minutes=3)
+        
+        super().save(*args, **kwargs)
+    
+    @staticmethod
+    def generate_otp_code():
```

Ruff (0.12.2)

Missing return type annotation for staticmethod `generate_otp_code`

Add return type annotation: `str`

(ANN205)